pyaedt/maxwell.py

```diff
@@ -1920,17 +1920,24 @@ def export_element_based_harmonic_force(
         self.odesign.ExportElementBasedHarmonicForce(output_directory, setup, freq_option, f1, f2)
         return output_directory
 
-    @pyaedt_function_handler
-    def edit_external_circuit(self, netlist_file_path, schematic_design_name):
+    @pyaedt_function_handler()
+    def edit_external_circuit(self, netlist_file_path, schematic_design_name, parameters=None):
         """
-        Edit the external circuit for the winding.
+        Edit the external circuit for the winding and allow edit circuit parameter.
```

Comment:
This isn't clear--I'm sure due to the desire to keep it brief. Should it be: "... and allow editing of the ``circuit`` parameter."

Do you think you can rewrite the description to make it clearer.

Reply:
done it!